keyboards/runner3680/3x6/3x6.c

```diff
@@ -0,0 +1 @@
+#include "runner3680.h"
```

Comment:
This should match the folder it's in. Eg: 

```suggestion
#include "3x6.h"
```